app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+require 'data_mapper'
+require 'bcrypt'
+require 'dm-validations'
+require 'dm-postgres-adapter'
+require 'dm-migrations'
+
+class User
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String
+  property :username, String
+  property :email, String
+  property :password_digest, Text
+
+  attr_accessor :password_confirmation
+  attr_reader :password
+
+  validates_confirmation_of :password, :confirm => :password_confirmation
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)